docs/scenarios/library-authoring/fastening-libraries-to-rc2.md

```diff
@@ -0,0 +1,68 @@
+# How to manage package dependency versions for .NET Core RC2
+
+By [Phillip Carter](https://github.com/cartermp)
+
+This article covers what you need to know about package versions for authoring libraries for .NET Core.
+
+## Glossary
+
+**Fasten** - In the context of authoring libraries, fastening dependencies means you are using the same "family" of packages released on NuGet for .NET Core RC2.
+
+**Metapackage** - A NuGet package that represents a set of NuGet packages.  They're organized in a way that makes sense for general library and app development.
+
+**Trimming** - The act of removing the packages you do not depend on from a metapackage.  The end result is that your library only depends on packages it uses.  See guidance in [How to trim your package dependencies](trimming.md).
+
+## Fasten your dependencies to RC2
+
+When creating libraries with .NET Core RC2, it's important that you fasten your dependencies to RC2.  This means two things.  First, you should only depend on packages marked as `rc2`.  Second, each package should come from the same build; that is, every package should have this pattern: `<package_version>-rc2-<build_number>`.  `build_number` is `3002700` for the `Microsoft.NETCore.App` metapackage and `24027` for all other packages.
+
+**Examples of packages fastened to RC2**
+
+`"System.Collections":"4.0.11-rc2-24027"`
+
+`"NETStandard.Library":"1.5.0-rc2-24027"`
+
+`"Microsoft.NETCore.App":"1.0.0-rc2-3002700"`
+
+**Examples of packages that are NOT fastened to RC2**
+
+`"System.Collections":"4.0.11-rc3-24021-00"`
+
+`"System.Net.Http":"4.1.0-rc2-24008"`
+
+`"System.Text.RegularExpressions":"4.0.10-rc3-24021-00"`
+
+### Why does this matter?
+
+We guarantee that if you fasten your dependencies to RC2, those packages will all work together.  There is no such guarantee if you use packages which aren't fastened to RC2.
+
+### Scenarios
+
+Although there is a big list of all packages and their versions released with .NET Core RC2, you may not have to look through it if you fall under certain scenarios.
```

Comment:
nit: wondering if the right thing to do here would be to change the subject. Is the developer the subject who falls under those scenarios or is it the project?